salt/utils/event.py

```diff
@@ -223,17 +226,54 @@ def __load_uri(self, sock_dir, node):
         )
         return puburi, pulluri
 
-    def subscribe(self, tag=None):
+    def subscribe(self, tag):
```

Comment:
Might the requirement that a person pass a `tag` arg break existing 3rd-party use of this function? (I'm sure there's a fair amount of that.)